frontend/src/app/components/button/Button.tsx

```diff
@@ -3,20 +3,37 @@ import clsx from 'clsx';
 
 import './Button.css';
 
-export type ButtonVariant = 'primary' | 'secondary' | 'tertiary';
+export type ButtonVariant = 'primary' | 'secondary' | 'tertiary' | 'icon';
 export type ButtonSize = 'small' | 'medium';
 
 type ButtonProps = ButtonHTMLAttributes<HTMLButtonElement> & {
   size?: ButtonSize;
   variant?: ButtonVariant;
+  active?: boolean;
 };
 
 export const Button = forwardRef<HTMLButtonElement, ButtonProps>(
-  ({ variant = 'primary', size = 'medium', className, ...props }, ref) => {
+  (
+    {
+      variant = 'primary',
+      size = 'medium',
+      className,
+      active = false,
+      ...props
+    },
+    ref,
+  ) => {
+    const variantCssClass = variant === 'icon' ? 'btn-icon' : variant;
```

Comment:
Had to add the `btn` prefix here. Otherwise, the styles conflict with the existing `.icon` class somewhere else in the app 😬 

This is a symptom of a much larger issue and way out of scope of this PR. If you have ideas on how we should better modularize / encapsulate our CSS, I'd love to chat with you.